helpdesk/views/public.py

```diff
@@ -100,7 +100,7 @@ def view_ticket(request):
     email = request.GET.get('email', None)
     # If there is no email address in the query string, get it from
     # the currently logged-in user
-    if email == '':
+    if not email:
```

Comment:
This looks like a bug fix that could be accepted upstream in a PR?

Reply:
Upstream does not grab the e-mail from the current user; they just default to an error if you don't submit a ticket ID and an e-mail.
Would it still make sense to have a similar PR for the upstream?

Reply:
That makes sense. Sounds like this is not something we want to submit upstream then. However, as we just discussed in person, it would probably make sense to add this note to the commit message for future reference. Thanks!